docs/content/components/nci-hero-cta.mdx

```diff
@@ -0,0 +1,77 @@
+---
+title: NCI Hero Image with CTA Component - NCI Design System
+---
+
+# Hero Image w/Button, Text Box & CTA Strip
+
+## When to use the Hero Image w/ Button, Text Box & CTA Strip variation
+
+- When there is a main call-to-action
+- When you need to include multiple quick links
+
+## When to consider something else
+
+- When you need to display groups of information or long chunks of text
+
+# Default NCI Hero w/Button & Text Box and CTA Strip
+
+```html
+<div class="nci-hero-cta">
+	<section class="nci-hero">
+		<picture class="nci-hero__image">
+			<source
+				media="(min-width: 1024px)"
+				srcset="../nci-hero-images/hero-widescreen.jpg"
+			/>
+			<source
+				media="(min-width: 880px)"
+				srcset="../nci-hero-images/hero-desktop.jpg"
+			/>
+			<source
+				media="(min-width: 640px)"
+				srcset="../nci-hero-images/hero-tablet-large.jpg"
+			/>
+			<source
+				media="(min-width: 480px)"
+				srcset="../nci-hero-images/hero-tablet.jpg"
+			/>
+			<source
+				media="(min-width: 320px)"
+				srcset="../nci-hero-images/hero-mobile-large.jpg"
+			/>
+			<img src="../nci-hero-images/hero-mobile.jpg" alt="" />
+		</picture>
+		<div class="nci-hero__cta-container">
+			<div class="nci-hero__cta nci-hero__cta--with-button">
+				<h2 class="nci-hero__cta-tagline">
+					NCI is the nation's leader in cancer research.
+				</h2>
+				<a href="https://www.cancer.gov" class="nci-hero__cta-button usa-button"
+					>Learn More</a
```

Comment:
We should probably demonstrate how 'Learn More' buttons should have screenreader text associated with it so that it's screen reader friendly.  You should also add a note about this in an accessibility section on this doc page.
e.g.
```
<a href="https://www.cancer.gov" class="nci-hero__cta-button usa-button">Learn More<span class="usa-sr-only"> about the National cancer Institute</span></a>
```